ElectronClient/app/gui/NoteContentPropertiesDialog.tsx

```diff
@@ -1,94 +1,83 @@
 import * as React from 'react';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 const { _ } = require('lib/locale.js');
-const { themeStyle, buildStyle } = require('../theme.js');
+const { themeStyle } = require('../theme.js');
 const DialogButtonRow = require('./DialogButtonRow.min');
 const Countable = require('countable');
 
 interface NoteContentPropertiesDialogProps {
 	theme: number,
-	text: string,
+	text: number,
+	lines: number,
```

Comment:
It's a minor thing but please can you name it "lineCount". This is because in many places in Joplin "lines" would refer to an array of lines.

Reply:
Actually forget about this, or you would need to rename words, characters, etc. to be consistent. It indeed makes sense to call it "lines" in this context.